alien4cloud-yorc-plugin/src/main/resources/slurm/resources/resources.yaml

```diff
@@ -55,6 +55,11 @@ node_types:
         type: string
         required: false
         description: Specify a name for the job allocation. The specified name will appear along with the job id.
+      account:
+        type: string
+        description: >
+           Charge resources used by this allocation to specified account.
```

Comment:
Could add like below for the job account property: May be mandatory according to configuration.

Reply:
I miss it. I'll add it in the slurm job refactoring